packages/jupyter/src/controls/NotebookToolbar.tsx

```diff
@@ -41,8 +41,8 @@ export function NotebookToolbar({ showLaunch = false }: { showLaunch?: boolean }
 
   if (computable)
     return (
-      <div className="sticky top-[60px] flex justify-end w-full z-20">
-        <div className="flex p-1 m-1 space-x-1 border rounded-full shadow border-stone-300 bg-white/80 dark:bg-stone-900/80 backdrop-blur">
+      <div className="sticky top-[60px] flex justify-end w-full z-20 pointer-events-none">
+        <div className="flex p-1 m-1 space-x-1 border rounded-full shadow pointer-events-auto border-stone-300 bg-white/80 dark:bg-stone-900/80 backdrop-blur">
```

Comment:
guessing this was to stop the pointer showing up over blank space in the rest of the `div` yeah?

Reply:
Yep!